timescaledb/getting-started/launch-timescaledb/launch-timescaledb-cloud.md

```diff
@@ -1,49 +1,38 @@
-# Launch your first instance
-
- we'll use Timescale Cloud to get you up
-and running quickly. Timescale Cloud combines the power and reliability
-of TimescaleDB with a fully managed, cloud-native experience that is easy to
-start and less expensive to operate.
-
-Creating a Timescale Cloud account is fast and simple, allowing you to explore
-TimescaleDB free for 30 days, no credit card required. We're just excited
-that you're ready to dive into the world of time-series data with TimescaleDB.
-
-<highlight type="tip">
-If you would prefer to set up TimescaleDB in your own environment and follow
-along with the rest of the _Getting Started_ tutorial, please see our documentation
-for [installing self-hosted TimescaleDB](/install/latest/self-hosted/).
-</highlight>
+# Launch your first instance of TimescaleDB in the Cloud
 
 ### Step 1: Create a Timescale Cloud account
 
 Sign up for Timescale Cloud by visiting [console.cloud.timescale.com][cloud-signup].
 
 Provide your full name, email address, and a strong password to start:
 
-<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tsc-signup.png" alt="Sign up for Timescale Cloud"/>
+<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tsc-signup-full.png" alt="Sign up for Timescale Cloud"/>
 
-You need to confirm your account by clicking the link you receive via
-email. If you don't receive this link, first check your spam folder
-and, failing that, [contact us][contact-timescale].
+You need to confirm your account by clicking on the link you receive via email. If you 
+don't receive this link, first check your spam folder and, failing that, [contact us][contact-timescale].
 
 ### Step 2: Create your first service
 
-After confirming your account, visit the
-[Timescale Cloud console][cloud-console] and log in with your credentials.
+After confirming your account, there should be a link that will redirect you to your 
```

Comment:
in the email I guess?

Reply:
I mention the email in line 11 12ish, I figure that would be enough?